src/emerge.cpp

```diff
@@ -396,13 +396,6 @@ int EmergeManager::getGroundLevelAtPoint(v2s16 p)
 // TODO(hmmmm): Move this to ServerMap
 bool EmergeManager::isBlockUnderground(v3s16 blockpos)
 {
-#if 0
-	v2s16 p = v2s16((blockpos.X * MAP_BLOCKSIZE) + MAP_BLOCKSIZE / 2,
-					(blockpos.Y * MAP_BLOCKSIZE) + MAP_BLOCKSIZE / 2);
-	int ground_level = getGroundLevelAtPoint(p);
-	return blockpos.Y * (MAP_BLOCKSIZE + 1) <= min(water_level, ground_level);
-#endif
-
 	// Use a simple heuristic; the above method is wildly inaccurate anyway.
```

Comment:
I'd remove the part after the semicolon in the comment since there is no "above method" anymore.

Reply:
```suggestion
	// Use a simple heuristic
```